Source/WebKit/Shared/libwpe/WebEventFactory.cpp

```diff
@@ -157,6 +157,41 @@ static inline short pressedMouseButtons(uint32_t modifiers)
     return buttons;
 }
 
+// FIXME: Double-click time should be at least 500ms (now it's 1s).
+// To fix it, it would be needed that libwpe provides milliseconds information in
+// wpe_input_pointer_event. Right now it only provides time in seconds.
+// See: https://github.com/WebPlatformForEmbedded/libwpe/blob/master/include/wpe/input.h#L79
+static inline double getDoubleClickTime()
```

Comment:
I think `time` in `wpe_input_pointer_evet` stores UNIX timestamps, which is seconds.

From `WebEventFactory.cpp`:

```
// WPE event time field is an uint32_t, too small for full ms timestamps since
// the epoch, and are expected to be just timestamps with monotonic behavior
// to be compared among themselves
```

https://searchfox.org/wubkat/source/Source/WebKit/Shared/libwpe/WebEventFactory.cpp#95

If I make `getDoubleClickTime()` return 500, which whould work if `time` is milliseconds, the two tests that are passing now are failing.

I think it would be useful to store milliseconds, microseconds or nanoseconds information as part of `wpe_input_pointer_evet`, probably as an extra field.

Reply:
That's only true for tests, it's a bug in EventSenderProxyLibWPE, I would say, that uses seconds for the synthesized events, but it's not the case for actual events.